roles/ceph-common/handlers/main.yml

```diff
@@ -91,13 +94,14 @@
   when:
     socketrgw.rc == 0 and
     ansible_distribution == 'Ubuntu' and
+    not ubuntu_use_systemd and
     rgw_group_name in group_names
 
 - name: restart ceph rgws
   command: /etc/init.d/radosgw restart
```

Comment:
I think we need to make sure that Ubuntu nodes that want systemd *do not* make this call, no?